x-pack/plugin/kql/src/yamlRestTest/resources/rest-api-spec/test/kql/50_kql_nested_fields_query.yml

```diff
@@ -0,0 +1,219 @@
+setup:
+  - requires:
+      capabilities:
+        - method: POST
+          path: /_search
+          capabilities: [ kql_query ]
+      test_runner_features: capabilities
+      reason: KQL query is not available
+
+  - requires:
+      "test_runner_features": "contains"
```

Comment:
The `reason` mentioned in the `requires` clause is for the kql capability.  
It would not make sense if tracking the `contains` feature in the case where we would merge it.